zmon_worker_monitor/builtins/plugins/scalyr.py

```diff
@@ -24,25 +25,30 @@ def create(self, factory_ctx):
         :param factory_ctx: (dict) names available for Function instantiation
         :return: an object that implements a check function
         """
-        return propartial(ScalyrWrapper, read_key=self.read_key)
+        return propartial(ScalyrWrapper, read_key=self.read_key, scalyr_region=self.scalyr_region)
 
 
 class ScalyrWrapper(object):
-    def __init__(self, read_key):
-        self.__numeric_url = 'https://www.scalyr.com/api/numericQuery'
-        self.__timeseries_url = 'https://www.scalyr.com/api/timeseriesQuery'
-        self.__facet_url = 'https://www.scalyr.com/api/facetQuery'
+    def __init__(self, read_key, scalyr_region=None):
+        if scalyr_region == 'eu':
+            self.__numeric_url = 'https://eu.scalyr.com/api/numericQuery'
```

Comment:
Suggestion:
We could define ``SCALYR_URL_PREFIX_US`` and ``SCALYR_URL_PREFIX_EU``

```python
SCALYR_URL_PREFIX_US = 'https://www.scalyr.com/api'
..


scalyr_prefix = SCALYR_URL_PREFIX_US
if scalyr_region == 'eu':
    scalyr_prefix = SCALYR_URL_PREFIX_EU

self.__numeric_url = '{}/numericQuery'.format(sclayr_prefix)
...
```